ansible/roles/test/files/helpers/change_mac.sh

```diff
@@ -7,7 +7,7 @@ INTF_LIST=$(ls /sys/class/net | grep -E "^eth[0-9]+$")
 for INTF in ${INTF_LIST}; do
     ADDR="$(cat /sys/class/net/${INTF}/address)"
     PREFIX="$(cut -c1-15 <<< ${ADDR})"
-    SUFFIX="$(printf "%02x" ${INTF##eth})"
+    SUFFIX="$(printf "%02x" $((${INTF##eth}%256)))"
```

Comment:
This part needs to be updated to avoid duplication when more than 256.

Reply:
Thank you for catching that! I will push the update shortly.

Reply:
Update pushed.